avm/src/daemon/ctrl.rs

```diff
@@ -107,8 +113,18 @@ async fn control_port(req: Request<Body>) -> Result<Response<Body>, Infallible>
   if cluster_secret.is_some() && !req.headers().contains_key(cluster_secret.as_ref().unwrap()) {
     // If this control port is guarded by a secret string, make sure there's a header with that
     // secret as the key (we don't care about the value) and abort otherwise
-    Ok(Response::builder().status(500).body("fail".into()).unwrap())
-  } else if TcpStream::connect("127.0.0.1:443").is_err() {
+    return Ok(Response::builder().status(500).body("fail".into()).unwrap());
+  }
+  match req.uri().path() {
+    "/health" => Ok(Response::builder().status(200).body("ok".into()).unwrap()),
+    "/clusterHealth" => handle_cluster_health(),
```

Comment:
Why are these different?

Reply:
`/health` tell us control port is ready and `clusterHealth` checks for 443 port abd all that logic

Reply:
/ health is being used to know when control port is active to send the `POST` to start the VM